clang/tools/libclang/CMakeLists.txt

```diff
@@ -135,6 +135,7 @@ endif()
 add_clang_library(libclang ${ENABLE_SHARED} ${ENABLE_STATIC} INSTALL_WITH_TOOLCHAIN
   OUTPUT_NAME ${output_name}
   ${SOURCES}
+  PARTIAL_SOURCES_INTENDED
```

Comment:
As noted below, I think we should always include DllMain.cpp (on Windows), and then I think we don't need this?